File: demo-app/app/src/javaApp/java/com/launchkey/android/authenticator/demo/ui/activity/CustomSecurityActivity.java
package com.launchkey.android.authenticator.demo.ui.activity;

import android.os.Bundle;
import androidx.annotation.Nullable;

import com.launchkey.android.authenticator.demo.R;
import com.launchkey.android.authenticator.demo.databinding.DemoActivitySecurityBinding;

public class CustomSecurityActivity extends BaseDemoActivity<DemoActivitySecurityBinding> {
    public CustomSecurityActivity() {
        super(R.layout.demo_activity_security);
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DemoActivitySecurityBinding.bind(findViewById(R.id.custom_sec_root));
        setSupportActionBar(binding.secToolbar);
    }
}

File: demo-app/app/src/kotlinApp/java/com/launchkey/android/authenticator/demo/ui/activity/CustomSecurityActivity.kt
package com.launchkey.android.authenticator.demo.ui.activity

import android.os.Bundle
import androidx.appcompat.widget.Toolbar
import androidx.viewbinding.ViewBinding
import com.launchkey.android.authenticator.demo.R

class CustomSecurityActivity : BaseDemoActivity<ViewBinding>() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.demo_activity_security)
        val toolbar = findViewById<Toolbar>(R.id.sec_toolbar)
        setSupportActionBar(toolbar)
    }
}